QueryBuilder/Helper.cs

```diff
@@ -25,36 +25,15 @@ public static bool IsNumber(object value)
 
         public static bool IsArray(object value)
         {
-            if (value == null)
-            {
-                return false;
-            }
-
             if (value is string)
-            {
```

Comment:
leaving the braces is more preferable https://softwareengineering.stackexchange.com/questions/2715/should-curly-braces-appear-on-their-own-line

Reply:
ok, i agree 😉, what your choice both below options?

```csharp
// A
if (statement) { /* do something */ }

// B
if (statement)
{
    /* do something */
}
```

In project, several guard clause are exists without braces. i thought i could get rid of them. 😅

Reply:
I think B is more appropriate, maybe we have to use a tool such StyleCop or something similar :)